pkg/collector/corechecks/servicediscovery/envs/envs.go

```diff
@@ -0,0 +1,86 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package envs provides target environment variables of interest.
+package envs
+
+import (
+	"fmt"
+)
+
+// targets is a collection of environment variables of interest.
+var targets = map[string]struct{}{
+	"PWD":                      {},
+	"DD_INJECTION_ENABLED":     {},
+	"DD_SERVICE":               {},
+	"DD_TAGS":                  {},
+	"DD_DISCOVERY_ENABLED":     {},
+	"GUNICORN_CMD_ARGS":        {},
+	"WSGI_APP":                 {},
+	"CORECLR_ENABLE_PROFILING": {},
+	"CATALINA_OPTS":            {},
+	"JAVA_TOOL_OPTIONS":        {},
+	"_JAVA_OPTIONS":            {},
+	"JDK_JAVA_OPTIONS":         {},
+	"JAVA_OPTIONS":             {},
+	"JDPA_OPTS":                {},
+	"SPRING_APPLICATION_NAME":  {},
+	"SPRING_CONFIG_LOCATIONS":  {},
+	"SPRING_CONFIG_NAME":       {},
+	"SPRING_PROFILES_ACTIVE":   {},
+}
+
+// GetExpectedEnvs - return list of expected env. variables for testing.
+func GetExpectedEnvs() ([]string, map[string]string) {
```

Comment:
if that's only for testing, let's move it to a dedicated file (envs_testutils.go) and add a build flag `//go:build test`

it will allows us ensure no-testing only code base enters to our production versions

Reply:
I've created separate file envs_testutils.go with a code dedicated for tests only